index.html

```diff
@@ -47,6 +47,148 @@ <h2 class="secondary-text">I'am a Software Developer</h2>
     </div>
   </section>
```

Comment:
Please add the background image present in this section
This is the figma design:
<img width="372" alt="Screenshot 2022-10-29 at 18 21 16" src="https://user-images.githubusercontent.com/48546992/198839800-10cc9c78-0d02-46f4-8142-9fab5c57669e.png">
Your design does not have this
<img width="1677" alt="Screenshot 2022-10-29 at 18 21 11" src="https://user-images.githubusercontent.com/48546992/198839817-3c3a786c-4fb2-4b75-a654-db0ed6eab552.png">

Reply:
I can see a background image in my design. See the image(https://user-images.githubusercontent.com/114238556/198841296-4029cc1d-662a-4c82-a3b8-04bee2b68aff.png))

Reply:
@Nyame-Wolf please help me in this matter.